src/platform/site-wide/header/components/LogoRow/styles.scss

```diff
@@ -12,6 +12,23 @@
   .profile-nav-contianer > div {
     padding-top: 0 !important;
   }
+
+  .sign-in-drop-down-panel-button {
+    border-top: none !important;
```

Comment:
Is there a way we can apply without using `!important`? Applies for all in the PR

Reply:
Nope, unfortunately

Reply:
All of this code for styling the user nav will be moot once I get the time to rewrite it for v2

Reply:
:sad-parrot: Just wanted to double check!

Reply:
totally! it was a great comment! 🔥 I am really looking forward to removing them, I am not a fan of `!important` 😂